deploy/flux-helm-release-crd.yaml

```diff
@@ -39,57 +39,41 @@ spec:
           required: ['chart']
           properties:
             releaseName:
-              description: The Helm release name. If not supplied, it will be generated by affixing the
```

Comment:
Why are these removed?

Reply:
Looks like make generate-deploy removes them, I have to add the description in the crd template.

Reply:
I wander why the CI didn't find this in #24